tests/manual/bundle_1_e2e_tls.yaml

```diff
@@ -0,0 +1,45 @@
+bundle: kubernetes
+applications:
+  alertmanager:
+    charm: ../../alertmanager-k8s_ubuntu-20.04-amd64.charm
+    series: focal
+    resources:
+      alertmanager-image: ghcr.io/canonical/alertmanager:0.25.0
+    scale: 1
+    trust: true
+  prometheus:
+    charm: prometheus-k8s
+    channel: edge
+    scale: 1
+    trust: true
+  local-ca:
+    charm: self-signed-certificates
+    channel: edge
+    scale: 1
+  external-ca:
+    charm: self-signed-certificates
+    channel: edge
+    scale: 1
+  traefik:
+    charm: traefik-k8s
```

Comment:
For now, this needs to be a local charm packed from https://github.com/canonical/traefik-k8s-operator/pull/245